src/ConnectFour/Domain/Game/Event/PlayerMoved.php

```diff
@@ -24,7 +24,8 @@ public function __construct(
         Point $point,
         Stone $stone,
         public readonly string $playerId,
-        public readonly string $nextPlayerId
+        public readonly string $nextPlayerId,
+        public readonly bool $abortable
```

Comment:
The business rule is pretty fix, and the properties would only toggle once in a game. So, duplicating the business logic on the client, rather than the server tells the client what to do is the more sane option in this scenario.

To get it right, it would need to expose `abortable` and `resignable`, because currently `resignable` is also calculated implicitly depending on `abortable` in client code.

Overall, this produce less code, less data with the downside of being a bit less DRY for the business logic.